iocore/dns/DNSEventIO.cc

```diff
@@ -0,0 +1,38 @@
+/** @file
+
+  A brief file description
```

Comment:
Better to omit @file if no actual description:  https://github.com/apache/trafficserver/blob/240025962cd85b89d50beec2579d114e84cd5782/include/shared/rpc/RPCRequests.h#L2

Reply:
Fixed